docs/guides/1230-persistent-cache.md

```diff
@@ -0,0 +1,219 @@
+install dependencies, build the binary and output it.
+
+Here's an example of a run
+
+```shell
+[✔] actions.build.container                                               338.0s
+[✔] client.filesystem.".".read                                              0.1s
+[✔] actions.build.container.export                                          0.1s
+[✔] client.filesystem."./output".write                                      0.2s
+```
+
+Indeed, Dagger has an ephemeral cache so if you rerun it, that shouldn't take
+that long.
+
+```shell
+[✔] actions.build.container                                                 2.9s
+[✔] client.filesystem.".".read                                              0.0s
+[✔] actions.build.container.export                                          0.0s
+[✔] client.filesystem."./output".write                                      0.1s
+```
+
+But if you stop the Buildkit daemon and remove its storage volume, cache will be lost and
+all actions will be executed again.
+
+:::info
+Now we have seen how ephemeral cache works, let's continue to understand how
+store cache in your local filesystem, so you can clean your Buildkit daemon without
+losing all your CI's cache.
+:::
+
+## Persistent cache in your local filesystem
+
+To store cache in your local filesystem, you don't need much effort : just
+run `dagger do build --cache-from type=local,mode=max,dest=<output folder>`
+
+:::tip
+Using `mode=max` argument will cache **all** layers from intermediate
+steps, with is really useful in the context of Dagger where you will have
+multiple steps to execute.  
+To only store the final layer, use `mode=min`.
+:::
+
+Here's an example that exports cache to a local directory at path `./storage`.
+
+```shell
+dagger do build --cache-to type=local,mode=max,dest=storage 
+# ...
+
+tree storage -L 1   
+storage
+├── blobs
+├── index.json
+└── ingest
+```
+
+As shown above, new directory has been created that contains cache artifacts from the run
+
+:::caution
+If you run different action in the plan, don't forget to store cache in different
+destination, so it will be not overwritten.
+:::
+
+To import the cache previously stored, you can use `--cache-from type=local,src=<cache folder>`.
+
+Here's an example, using a new buildkit daemon
+
+```shell
+# Down buildkit daemon
+docker container stop dagger-buildkitd && docker container rm dagger-buildkitd && docker volume rm dagger-buildkitd
+
+# Import cache on rebuild
+dagger do build --cache-to type=local,mode=max,dest=storage --cache-from type=local,src=storage
+[✔] actions.build.container                                                 2.3s
+[✔] client.filesystem.".".read                                              0.1s
+[✔] actions.build.container.export                                          0.0s
+[✔] client.filesystem."./output".write                                      0.4s
+```
+
+:::info
+In this part, we have how to keep cache in a local filesystem, if you want
```

Comment:
```suggestion
In this part, we have learned how to export and import cache using a local filesystem, if you want
```

Reply:
Fixed in https://github.com/dagger/dagger/pull/2035/commits/1651d359cfc25c6d69b505ae6103eeb90a06dc66